master/internal/task/allocation.go

```diff
@@ -549,6 +549,13 @@ func (a *Allocation) ResourcesStateChanged(
 	case sproto.Starting:
 		a.setMostProgressedModelState(model.AllocationStateStarting)
 	case sproto.Running:
+		if a.resources[msg.ResourcesID].Started != nil {
```

Comment:
😔 I wish our entire actor state machine system was designed around a possibility of duplicate messages.

Reply:
yeah..